`anstream::auto::current_choice` uses `ColorChoice` but isn't behind `auto` feature flag

resulting in

```
error[E0433]: failed to resolve: use of undeclared type `ColorChoice`
   --> src/auto.rs:141:44
    |
141 |             StreamInner::PassThrough(_) => ColorChoice::AlwaysAnsi,
    |                                            ^^^^^^^^^^^ use of undeclared type `ColorChoice`

error[E0433]: failed to resolve: use of undeclared type `ColorChoice`
   --> src/auto.rs:142:38
    |
142 |             StreamInner::Strip(_) => ColorChoice::Never,
    |                                      ^^^^^^^^^^^ use of undeclared type `ColorChoice`

error[E0412]: cannot find type `ColorChoice` in this scope
   --> src/auto.rs:139:37
    |
139 |     pub fn current_choice(&self) -> ColorChoice {
    |                                     ^^^^^^^^^^^ not found in this scope
```

--- a/crates/anstream/src/auto.rs
+++ b/crates/anstream/src/auto.rs
@@ -136,6 +136,7 @@ where
     ///
     /// This doesn't report what is requested but what is currently active.
     #[inline]
+    #[cfg(feature = "auto")]
     pub fn current_choice(&self) -> ColorChoice {
         match &self.inner {
             StreamInner::PassThrough(_) => ColorChoice::AlwaysAnsi,

